.github/workflows/contributors-record.yml

```diff
@@ -0,0 +1,32 @@
+# Give your test a name!
+name: ContributorsRecord
+
+# Decide when to run the tests
+#
+# This configuration sets the test to run on pushes to master
+# and on pull requests that are opened to master
+on:
+  push:
+    branches:
+      - master
```

Comment:
Do we really want to run this on master? Or is the idea that the commit will be added to open PRs that will then get merged?

Reply:
This should run on PRs - thank you for pointing. I guess I did not fully understand what parameters branches could be given.

Reply:
I think the config you want is:

```
on:
  pull_request:
    branches:
      - master
```

This will run on PRs that are opened with `master` as the target branch :)

Reply:
I am changing the bot to run weekly instead on all the PRs (mostly unrelated). 🤞